budgie-fuzzyclock/fuzzyclock/FuzzyClockApplet.vala

```diff
@@ -27,46 +27,46 @@ public const string CALENDAR_MIME = "text/calendar";
 
 public class FuzzyClockApplet : Budgie.Applet
 {
-    protected const string[] hours = {
-        "midnight",
-        "one",
-        "two",
-        "three",
-        "four",
-        "five",
-        "six",
-        "seven",
-        "eight",
-        "nine",
-        "ten",
-        "eleven",
-        "noon",
-        "thirteen",
-        "fourteen",
-        "fifteen",
-        "sixteen",
-        "seventeen",
-        "eighteen",
-        "nineteen",
-        "twenty",
-        "twenty-one",
-        "twenty-two",
-        "twenty-three",
+    protected string[] hours = {
+        _("midnight"),
+        _("one"),
+        _("two"),
+        _("three"),
+        _("four"),
+        _("five"),
+        _("six"),
+        _("seven"),
+        _("eight"),
+        _("nine"),
+        _("ten"),
+        _("eleven"),
+        _("noon"),
+        _("thirteen"),
+        _("fourteen"),
+        _("fifteen"),
+        _("sixteen"),
+        _("seventeen"),
+        _("eighteen"),
+        _("nineteen"),
+        _("twenty"),
+        _("twenty-one"),
+        _("twenty-two"),
+        _("twenty-three"),
     };
 
-    protected const string[] rules = {
-        "%s-ish",
-        "a bit past %s",
-        "ten past %s",
-        "quarter after %s",
-        "twenty past %s",
-        "almost half-past %s",
-        "half-past %s",
-        "twenty-five 'til %s",
-        "twenty 'til %s",
-        "quarter 'til %s",
-        "ten 'til %s",
-        "almost %s",
+    protected string[] rules = {
```

Comment:
Please split these into individual lines with a comment "# TRANSLATORS: example of 'ten past' is 5.10PM" so that translators understand the context